src/RedirectConfig.js

```diff
@@ -34,6 +35,11 @@ class RedirectConfig extends SchemaDerivedConfig {
       },
     });
   }
+
+  async match(path) {
+    // eslint-disable-next-line max-len
+    return this.redirects.reduce((matched, redirect) => matched || (redirect instanceof Redirect ? redirect.match(path) : null), null);
```

Comment:
:-) I reverted it back to `reduce`. I was wrong with `find`, as it doesn't return the first predicate result but the first item.

Reply:
Btw, do you want to approve the PR or use #327 instead?

Reply:
I would use this one first....